src/web/views/guidance/en/report-a-change.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "../../templates/guidance.njk" %}
+
+{% block guidanceContent %}
+<h2 class="govuk-heading-l">Report a change</h2>
+
+<p class="govuk-body">You must always report a change in your circumstances or personal details. </p>
+
+<p class="govuk-body">Changes you need to report include:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>a birth</li>
+  <li>a new pregnancy</li>
+  <li>change of name</li>
+  <li>change of contact details</li>
+</ul>
+
+<hr class="govuk-section-break govuk-section-break--m">
+
+<p class="govuk-body">This is to make sure you receive:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>the right amount of money</li>
+  <li>emails and letters about payments and changes</li>
+  <li>replacement cards</li>
+</ul>
+
+<hr class="govuk-section-break govuk-section-break--m">
+
+<p class="govuk-body">If you’re claiming any benefits you must also tell the <a class="govuk-link" href="https://www.gov.uk/report-benefits-change-circumstances"> benefit agencies</a> about these changes. </p>
+
+<div class="govuk-warning-text">
```

Comment:
I think there is a Nunjucks macro for this https://design-system.service.gov.uk/components/warning-text/